roles/internal/webserver-app/tasks/solr.yml

```diff
@@ -6,12 +6,12 @@
 - name: Set default solr server host from hostvars
   command: "{{ drush_path }} --root {{ drupal_core_path }} -y config-set search_api.server.default_solr_server backend_config.connector_config.host {{ hostvars[groups['solr'][0]].ansible_host  }}"
   register: set_search_api_config_host
-  changed_when: "'Do you want to update' in set_search_api_config.stdout"
+  changed_when: "'Do you want to update' in set_search_api_config_host.stdout"
 
 - name: Set default solr server to point to first core
   command: "{{ drush_path }} --root {{ drupal_core_path }} -y config-set search_api.server.default_solr_server backend_config.connector_config.core {{ solr_cores[0] }}"
   register: set_search_api_config_core
-  changed_when: "'Do you want to update' in set_search_api_config.stdout"
+  changed_when: "'Do you want to update' in set_search_api_config_core.stdout"
```

Comment:
Looking back at https://github.com/Islandora-Devops/islandora-playbook/pull/126/files#diff-5bddb401e18178b7584b51cd9831eaa348385079b2224a85e3db8fa2412fc547L5-R15 ... I believe this is the intent, as otherwise, it seems that the `set_search_api_config` bit would've been being registered by the "Matomo` setup, in the `webserver-app/tasks/drupal.yml` bit?

Reply:
Yeah that seems like a typo in the Matomo stuff.